decorator.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+require_relative('./nameable')
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
Kindly use the 'capitalize' method instead of 'upcase'. We only need the first letter to be uppercase. The rest of the name should be in lowercase. The method you have used returns all the letters in uppercase. Kindly change this.

Reply:
@muneebulrehman Thank you for the review :heart: